src/XrdPosix/XrdPosixDir.hh

```diff
@@ -69,6 +70,11 @@ static int         dirNo(DIR *dirP)  {return *(int *)dirP;}
 
        DIR        *Open();
 
+       // Return the stat info corresponding to the current directory entry
+       // On error, returns -errno; otherwise, returns 0 and stores a reference
```

Comment:
Hmm, it stores a pointer not a reference to the buffer. Please correct the comment.

Reply:
:) I think I was being overly pedantic - a pointer is a type of reference (unfortunately, a C++ reference is also a reference...).  Happy to change it.